src/components/seeds/lab/NewCutTest.tsx

```diff
@@ -59,8 +59,8 @@ export default function NewCutTest(props: Props): JSX.Element {
           <Grid container spacing={4}>
             <Grid item xs={12}>
               <TextField
-                id='cutTestSeedsFilled'
-                value={record.cutTestSeedsFilled}
+                id='viabilityTests_seedsFilled'
+                value={record.viabilityTests_seedsFilled}
```

Comment:
I'm not sure if these changes are correct or are even needed?

Reply:
We are not using this file anymore

Reply:
Got it, thanks

Reply:
Ok I left the changes in this file since [Karthik's PR](https://terraformation.atlassian.net/browse/SW-1449) is removing this file anyways.

Reply:
Yeah, I will merge my stuff after this PR goes in.